apollo-configservice/pom.xml

```diff
@@ -64,6 +64,10 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>com.sun.jersey.contribs</groupId>
+			<artifactId>jersey-apache-client4</artifactId>
```

Comment:
The function of `jersey-apache-client4` is ?

And in `apollo-adminservice`, its scope is `test`
```xml
		<dependency>
			<groupId>com.sun.jersey.contribs</groupId>
			<artifactId>jersey-apache-client4</artifactId>
			<scope>test</scope>
		</dependency>
```

So it should be test scope too here?

Reply:
jersey-apache-client4 was a dependency of eureka server, but was removed recently, so we have to add it back, see https://github.com/spring-cloud/spring-cloud-netflix/issues/3953

apollo-adminservice doesn't need it in runtime, but requires it in test as we need to start a eureka server in test mode